pkg/fedora/tarsnap.spec

```diff
@@ -0,0 +1,59 @@
+Name:           tarsnap
+Version:        1.0.36.1
+Release:        2%{?dist}
+Summary:        Secure, efficient online backup service
+
+License:        Proprietary
+URL:            http://www.tarsnap.com/
+Source0:        http://www.tarsnap.com/download/tarsnap-autoconf-%{version}.tgz
+
+BuildRequires:  gcc
+BuildRequires:  pkgconfig(bash-completion)
+BuildRequires:  pkgconfig(bzip2)
+BuildRequires:  pkgconfig(ext2fs)
+BuildRequires:  pkgconfig(libcrypto)
```

Comment:
The `configure` script checks for a few optional dependencies not listed on the Tarsnap website.  I looked again, and it seems libattr is not actually needed for xattr support; tarsnap will use the syscall wrappers in glibc.  I'll update the specfile.  libacl does seem to be required for ACL support, though, and I'm guessing bzip2 and lzma are used for `@archive-file` support.  Either way, it'd be good to get the configure script and website in sync.

`pkg/debian/control` includes all of those dependencies except liblzma.  So, if the above is correct, liblzma should be added there and libattr should be dropped.